modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -82,7 +82,23 @@
     /**
      * A mode when data nodes throttle update rate regarding to DR sender load
      */
-    DR_DATA_NODE_SMART_THROTTLING(19);
+    DR_DATA_NODE_SMART_THROTTLING(19),
+
+    /**
+     * Support rolling upgrade status task for Web Console.
+     */
+    WC_ROLLING_UPGRADE_STATUS(20),
+
+    /**
+     * Support for chain parameter in snapshot delete task for Web Console.
+     */
+    WC_SNAPSHOT_CHAIN_MODE(21),
```

Comment:
One more comment - you must add unit test for new features

Reply:
We already have e2e test